src/control/controlframeposproxy.cpp

```diff
@@ -0,0 +1,16 @@
+#include "control/controlframepos.h"
+#include "moc_controlframeposproxy.cpp"
+
+ControlFramePosProxy::ControlFramePosProxy(const ConfigKey& key, QObject* pParent)
+        : ControlProxy(key, pParent) {
+    connect(this,
+            &ControlFramePosProxy::valueChanged,
+            this,
+            &ControlFramePosProxy::slotValueChanged,
+            Qt::DirectConnection);
+}
+
+void ControlFramePosProxy::slotValueChanged(double value) {
+    const auto position = mixxx::audio::FramePos::fromEngineSamplePosMaybeInvalid(value);
+    emit positionChanged(position);
```

Comment:
Ah git it, my original comment applies here. 
Please consider to make this a virtual function of ControlProxy. The base class can than emit valueChanged and this version can positionChanged(position).

You need also your own connectValueChange() function to connect the different typed slot.

Reply:
It looks like receiving updates is not yet used.
How about merge the PollingControlProxy() branch first and inherit from the PollingControlProxy() or reimplement it for FramePosition.